<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    padding: 8px;
    border-radius: 5px;
}
QLineEdit {
    border: 1px solid #95a5a6;
    padding: 4px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="user_input"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    padding: 8px;
    border-radius: 5px;
}
QLineEdit {
    border: 1px solid #95a5a6;
    padding: 4px;
}
</string>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pass_input">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="login_button">
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
